airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/brands_report.py

```diff
@@ -42,6 +42,9 @@
         "attributedUnitsOrderedNewToBrandPercentage14d",
         "unitsSold14d",
         "dpv14d",
+        "attributedBrandedSearches14d",
+        "keywordId",
+        "searchTermImpressionShare",
```

Comment:
I see that you added the missing metrics that are defined in the documentation https://advertising.amazon.com/API/docs/en-us/reporting/v2/report-types. But for some record types, the current code has redundant metrics. For example, in this list, there are `targetId`, `targetingExpression`, `targetingText`, `targetingType`. However, the documentation doesn't indicate they are available for this particular record type. Does the report return any values for them? If not, such metrics that aren't listed in the documentation should be removed.

Please check all the report types, all record types, and all the metrics. The code should exactly reflect the documentation except when the documentation is wrong, then follow the actual behavior of the reporting system.

Reply:
@monai - Thanks for your feedback. I am working on it and will update the PR soon.

Reply:
@monai - I updated the PR and deleted the metrics that are no longer available or supported. There are also certain fields indicated in the documentation but not supported, such as 'attributedKindleEditionNormalizedPagesRoyalties14d' and 'attributedKindleEditionNormalizedPagesRead14d' for Sponsored products, which I have not included in the metrics.

Please review. Thank you.